var/spack/repos/builtin/packages/ucx/package.py

```diff
@@ -58,6 +62,8 @@ class Ucx(AutotoolsPackage, CudaPackage):
     depends_on('gdrcopy@1.3', when='+gdrcopy')
     conflicts('+gdrcopy', when='~cuda',
               msg='gdrcopy currently requires cuda support')
+    depends_on('knem@1.1.4', when='+knem')   # only v1.1.4+ contains knem.pc
```

Comment:
```suggestion
    depends_on('knem@1.1.4:', when='+knem')   # only v1.1.4+ contains knem.pc
```
The colon here makes this equivalent to 1.1.4+